### PR TITLE
Fix document display for demo

### PR DESCRIPTION
**Pull request by Jasopaum**

### CODE REVIEW

**Review by tevariou — APPROVED** [review]
👍

File: src/services/cohortInfos.ts [hunk 1]
```diff
@@ -486,7 +486,8 @@ const fetchDocuments = async (
       ),
       search !== '?'
         ? api.get<FHIR_API_Response<IDocumentReference>>(
-            `/DocumentReference?_sort=${_sortDirection}${sortBy}${searchByGroup}${docTypesFilter}${ndaFilter}${dateFilter}&_summary=count`
+            // `/DocumentReference?_sort=${_sortDirection}${sortBy}${searchByGroup}${docTypesFilter}${ndaFilter}${dateFilter}&_summary=count`
```
Comment (tevariou) [inline src/services/cohortInfos.ts:493]: rm ? or a comment on why sortBy doesn't work?

Comment (Jasopaum) [reply src/services/cohortInfos.ts:493]: I'll add a comment 👍

File: src/services/cohortInfos.ts [hunk 2]
```diff
@@ -479,10 +479,12 @@ const fetchDocuments = async (
       // api.get<FHIR_API_Response<IComposition>>(
       //   `/Composition?facet=cloud&size=0&_sort=${_sortDirection}${sortBy}&status=final${elements}${searchByGroup}${search}${docTypesFilter}${ndaFilter}${dateFilter}`
       // ),
+      // When trying to sort on the `date` search param, the api returns 0 resources.
+      // See https://github.com/arkhn/Cohort360/issues/113
       api.get<FHIR_API_Response<IDocumentReference>>(
         `/DocumentReference${search}_count=10000&_getpagesoffset=${
           page ? (page - 1) * 20 : 0
-        }&_sort=${_sortDirection}${sortBy}${searchByGroup}${docTypesFilter}${ndaFilter}${dateFilter}`
+        }&_sort=${_sortDirection}${searchByGroup}${docTypesFilter}${ndaFilter}${dateFilter}`
```
Comment (Jasopaum) [inline src/services/cohortInfos.ts]: What? Don't you need to also remove `&_sort=${_sortDirection}`?

**Review by simonvadee — APPROVED** [review]

**Pull request merged**